app/filtering.js

```diff
@@ -406,6 +414,19 @@ function onBlockedInTor (details, muonCb) {
   }
 }
 
+function onBlockedOutsideTor (details, muonCb) {
```

Comment:
[minor] to avoid code duplication, best to refactor this and `onBlockedInTor` into a single method that takes an argument indicating why it was blocked

Reply:
done